packages/xo-web/src/xo-app/tasks/index.js

```diff
@@ -286,19 +286,23 @@ export default class Tasks extends Component {
       <Page header={HEADER} title={`(${nTasks}) ${formatMessage(messages.taskPage)}`}>
         <Container>
           <Row className='mb-1'>
-            <Col mediumSize={8}>
+            <Col mediumSize={7}>
               <SelectPool multi onChange={this.linkState('pools')} />
             </Col>
             <Col mediumSize={4}>
               <div ref={container => this.setState({ container })} />
             </Col>
+            <Col mediumSize={1}>
+              <div ref={itemsPerPage => this.setState({ itemsPerPage })} />
```

Comment:
Can we use [callback-refs](https://reactjs.org/docs/refs-and-the-dom.html#callback-refs)?